pages/common/fossa.md

```diff
@@ -0,0 +1,24 @@
+# fossa
+
+> Fossa CLI - Integrate realtime license audits, vulnerability scans and reports about dependencies licenses.
+> Homepage: <https://github.com/fossas/fossa-cli>.
+
+- Initialize a {{.fossa.yml}} configuration file:
```

Comment:
```suggestion
- Initialize a `.fossa.yml` configuration file:
```
What @waldyrious meant here was backticks (`), not curly braces.

Reply:
My bad. Thanks.